autoload/go/coverage.vim

```diff
@@ -46,7 +46,7 @@ function! go#coverage#Buffer(bang, ...) abort
 
   if go#util#has_job()
     call s:coverage_job({
-          \ 'cmd': ['go', 'test', '-coverprofile', l:tmpname],
+          \ 'cmd': ['go', 'test', '-coverprofile', l:tmpname] + a:000,
```

Comment:
What if I don't pass anything? What is the value of a:000? How does coverage look like for a single function?